packages/settings/src/schema/central.ts

```diff
@@ -31,6 +31,97 @@ export const centralSettings = {
         },
       },
     },
+    sync: {
+      description: 'Settings related to sync',
+      highRisk: true,
+      properties: {
+        streaming: {
+          properties: {
+            enabled: {
+              description: 'Use streaming endpoints',
+              type: yup.boolean(),
+              defaultValue: false,
```

Comment:
It would be good to turn this on in at least one environment, else that code is sitting dormant, which is both pointless and risky if we were to turn it on later or under duress without battle testing
@passcod perhaps demo to start with?